### ISSUE TITLE
Investigate and implement to use xcat2.github.io 

### ISSUE THREAD

**Issue by bybai** [xcat2/xcat.org#118]
Use this task to trace "look into if we can use xcat2.github.io"

**Comment by bybai** [comment]
I got answers from github:
At the moment our repository xcat2/xcat.org is set up as a Project Page. If we set it up as an Organization Page, it's url would be http://xcat2.github.io.
We just need to change the repository name to be xcat2.github.io and make sure the files we want to deploy are on the master branch.
More information about Project Pages and Organization Pages can be found here:
https://help.github.com/articles/user-organization-and-project-pages/#user-and-organization-pages-sites

**Comment by bybai** [comment]
Hi @whowutwut,  
I read the github  Project Pages and Organization Pages. "xcat2" is ourt organization name. "xcat-core" and "xcat.org" are our project name. Github pages has schema to generate organization pages and project pages.

Based on above user docs, to create an Organization Pages site, name the repository using the naming scheme <orgname>.github.io , so our organization url is "xcat2.github.io" ,  but "xcat2.github.io" is 404 now.
A Project Pages site for an organization account is available at http(s)://<orgname>.github.io/<projectname>. In the past, we put all the code under the xcat2/xcat.org/, since it is a project page, the published link is "http://xcat2.github.io/xcat.org/". We have tested it.

If our organization name is "xcat-core", we can have "xcat-core.github.io". but now, our organization name is "xcat2".

Also, xcat2/xcat.org is now managing xcat.org website, so we cannot rename xcat2/xcat.org to xcat2/xcat2.github.io.

So I asked Ertao to help create new repo xcat2.github.io, and I created pull request here.
I want to verify if our organization page site works well. And the new page is better than current 404 page.

Here coming another question for future: if we want to keep xcat2.github.io and xcat.org both  or only one of them (here we can assume they have the same content). **???**
Now, I list the different choice and results:
1. only use xcat.org ---- current xcat2/xcat.org repo is enough, no need to use xcat2/xcat2.github.io 
2. only use xcat2.github.io ---- only need to keep xcat2/xcat2.github.io repo
3. both xcat2.github.io and xcat.org ---- use xcat2/xcat2.github.io repo, let xcat.org "git pull" from xcat2/xcat2.github.io

My opinion is to keep them both. It is no harm for us to keep both of them. What is your opinion?
